Apollo.xcodeproj/project.pbxproj

```diff
@@ -641,6 +642,7 @@
 				9FCE2D171E6C259B00E34457 /* Starship.graphql */,
 				9FCE2D0C1E6C259B00E34457 /* CreateReviewForEpisode.graphql */,
 				9FCE2D0A1E6C258A00E34457 /* API.swift */,
+				F8E9D8AE22B2492C0065DA98 /* schema.json */,
```

Comment:
any idea what schema was added here that wasn't already in there?

Reply:
This is schema.json in Starwar folder. 
Because schema.json in Github folder was also added to the project.
Just sync up the practice by also add this Starwar schema.json to the project.

Reply:
ahhhh got it, thanks!